src/runtime/contrib/cublas/cublas.cc

```diff
@@ -342,12 +434,33 @@ TVM_REGISTER_GLOBAL("tvm.contrib.cublas.matmul")
     }
 });
 
-TVM_REGISTER_GLOBAL("tvm.contrib.cublas.batch_matmul")
+#if CUDART_VERSION >= 10010
+TVM_REGISTER_GLOBAL("tvm.contrib.cublaslt.matmul")
 .set_body([](TVMArgs args, TVMRetValue* ret) {
     DLTensor* A = args[0];
-    DLTensor* C = args[2];
 
+    CuBlasThreadEntry* entry_ptr = CuBlasThreadEntry::ThreadLocal();
+
+    TryEnableTensorCore(entry_ptr->handle);
+
+    int version;
+    CHECK_CUBLAS_ERROR(cublasGetVersion(entry_ptr->handle, &version));
+    if (TypeMatch(A->dtype, kDLInt, 8) && version >= 10100) {
```

Comment:
version >= 10100 is no longer necessary. I would remove this if block and replace it with
```
CHECK(TypeMatch(A->dtype, kDLInt, 8), "Expects dtype to be int8");
cublasLtHandle_t ltHandle;
CHECK_CUBLAS_ERROR(cublasLtCreate(&ltHandle));
CallLtIgemm(args, ret, ltHandle);
CHECK_CUBLAS_ERROR(cublasLtDestroy(ltHandle));
```